stable/cert-manager/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: cert-manager
-version: v0.4.0
+version: v0.4.1
```

Comment:
Please bump to 1.0.0 . We had a call yesterday in the charts meeting we the approach will be all stable charts which is in  version 0.x.x should be at least 1.0.0

Cc @davidkarlsen @mattfarina

Reply:
If that's the case - we'll need to move cert-manager out of the 'stable' repository, as we are not currently committed to the API surface exposed in values.yaml between minor versions.

This is probably a conversation better had elsewhere though 😄